_maps/RandomRuins/SpaceRuins/the_faceoff.dmm

```diff
@@ -877,16 +861,15 @@
 /area/ruin/space)
 "HI" = (
 /obj/structure/closet/syndicate,
-/obj/item/clothing/mask/gas/syndicate,
 /obj/item/clothing/under/syndicate,
-/obj/item/clothing/suit/space/syndicate/black/red{
-	pixel_y = -4
-	},
 /obj/item/clothing/head/helmet/space/syndicate/black/red,
 /obj/item/pickaxe,
 /obj/effect/decal/cleanable/fuel_pool,
 /obj/effect/decal/cleanable/fuel_pool,
 /obj/item/flashlight/lantern,
+/obj/item/clothing/suit/space/orange,
+/obj/item/clothing/head/helmet/space/orange,
+/obj/item/clothing/mask/breath,
```

Comment:
I would prefer to figure out something else we can put there than to just remove it. What would be your suggestion instead of the black and red hardsuits? There are other colors we can use that are flagged for syndicate, or I could replace them with mining outfits, since it's a mining locker.

Reply:
Mining suits might be reasonable. Tell you what, I'll make a color appropriate explorer suit once I'm at my pc and send ya the dmi.

The orange suit sprites are currently used in two places. Emergency suits and as a narrative element for the xeno gateway mission so some Syndicate spies can pretend to be wearing a Nanotrasen suit.

Reply:
I very much so approve of the explorer suit change as those lockers are intended to be storage for the miners who hollowed out the asteroid. Not sure how I feel about removing the syndicate mask as its quite a common and easily obtainable item through both cargo and smugglers satchels.